### DISCUSSION TITLE
New UI ignores Default download directory and does not allow editing

### DISCUSSION THREAD

**Discussion by kidslow** [aclap-dev/video-downloadhelper#1229] [Help and Q&A]
### Your question

Firefox 115.4.0esr (64-bit)
MacOs 10.14.6

- Switch to the New UI.
- Try to edit download directory. Nothing happens. Cannot change. Remains as ~/dwhelper
- Switch to the Old UI.
- Update download directory to other than ~/dwhelper (e.g. ~/Downloads or ~/Documents/dwhelper ... etc.)
- Enable New UI
- Download
- Notice that it goes to ~/dwhelper
- Switch to the Old UI.
- Notice it is still configured to updated location from the default.

### Addon & CoApp version and details

Video DownloadHelper BETA 9.1.0.41 (beta)
Found companion app: VdhCoApp 2.0.19

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

It is possible you are expecting a behavior that VDH no longer adheres to.  You might want to read these discussions: #665, #744, #814.

On the other hand, there is a cutoff on the version of Mac on which the current CoApp works.  I don't know what version that is but you, as a Mac user, should be aware of what that version is.  I think you can figure it out from the CoApp references in our Wiki, a link to which appears at the top of every page of this forum.  It may also be, as a knock-on effect of any restriction on the version of the CoApp, that you might be forced to use a version of VDH older than 9.  Again, I am not a Mac user.  Never have been.  Know diddly squat about Mac.  So it's more on your shoulders to educate yourself about what does & does not work on old versions of Mac.

**Comment by kidslow** [comment]
Turns out that "educating" oneself on what does and does not work on an old version of Mac involves finding broken things when they appear broken  I am expecting functional behavior to be consistent with the interface. That is not an unreasonable expectation by a user. Seems broken to me but the workaround is to not use the new UI. Thought maybe someone else might benefit from knowing, which is why I reported it.
